live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -84,3 +84,25 @@
     dest: '/usr/src/linux-headers-{{ item }}/vmlinux'
     state: link
   loop: '{{ kernel_versions.stdout_lines }}'
+
+#
+# The PCP framework which provides `dstat` required by `delphix-platform`
+# is not needed on the appliance. We disable and mask the services that
+# are part of the PCP framework. While removing the package would be
+# a better option, it is not possible due to the dependency on `dstat`.
+#
+- name: Disable and mask the PCP services
```

Comment:
seb commented in slack.. but to address initial install and upgrade, this should likely be in the delphix-platform's ansible logic..